src/settings/Settings.js

```diff
@@ -158,6 +165,27 @@ export const SETTINGS = {
         displayName: _td("Show info about bridges in room settings"),
         default: false,
     },
+    "font_size": {
+        displayName: _td("Font size"),
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        default: 16,
+        controller: new FontSizeController(),
+    },
+    "font_size_min": {
+        displayName: _td("Min font size"),
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        default: 14,
+    },
+    "font_size_max": {
+        displayName: _td("Max font size"),
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        default: 24,
+    },
+    "useCustomFontSize": {
+        displayName: _td("Custom font size"),
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        default: false,
+    },
```

Comment:
I know we're horribly inconsistent, but please name all settings as `lowerCamelCase` instead of underscored.